build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
         classpath 'io.github.gradle-nexus:publish-plugin:1.1.0'
         classpath 'org.jetbrains.dokka:dokka-gradle-plugin:1.4.32'
+        classpath 'androidx.navigation:navigation-safe-args-gradle-plugin:2.3.5'
```

Comment:
Did you intend to add this out here in the main build gradle?

Reply:
The [official docs](https://developer.android.com/guide/navigation/navigation-pass-data) say to add it to the top level build.gradle file, and I couldn't get it to work any other way. I don't think it makes a difference to the SDK itself since none of those classes are being used there.